test/test_tfrecord.py

```diff
@@ -24,6 +24,15 @@
     TFRecordLoader,
 )
 
+try:
+    import google.protobuf as _protobuf
+
+    del _protobuf
+    HAS_PROTOBUF = True
+except ImportError:
+    HAS_PROTOBUF = False
+skipIfNoPROTOBUF = unittest.skipIf(not HAS_PROTOBUF, "no google protobuf")
+
```

Comment:
The changes in `test/test_tfrecord.py` are introduced because `protobuf` should be an optional dependency for TorchData test.